### PR TITLE
Updates to Collections show page UI/layout

### LINKED ISSUE #1678: Collections: Make admin show page match mockup
### Descriptive summary

Make the admin show page match the mockup.  The current UI implementation does not.

### Implementation

View files for the admin show page are located at...

[app/views/hyrax/dashboard/collections](https://github.com/samvera/hyrax/tree/collections-sprint/app/views/hyrax/dashboard/collections)

### Mockup

admin-collection-show

### Related UI work

#1427 - PR #1562 - Add content to dashboard collections show page
#1552 - PR #1632 - Add new work to collection from collection show page
#1586 - PR #1629 - Show page should only provide Remove action for works and sub-collections

### Related work

| Collections feature | Issue |
| ------------------------ | ----- |
| Routes                   | #1380 |
| Controller               | # |
| Model                    | #1383 |
| View - Index            | #1417 |
| View - Show - Admin      | #1427 |
| View - Show - Landing    | #1428 |
| View - New/Edit          | #1374 |
| Partial -  Description   | #1375 |
| Partial -  Branding      | #1376 |
| Partial -  Relationships | #1377 |
| Partial -  Sharing       | #1378 |
| Partial -  Discovery     | #1379 |

### PR DESCRIPTION
**Pull request by adamjarling**
Partially fixes #1678  ; 

Updates to the layout and UI of Collections show page, per latest wire frame attached to #1678 

See https://github.com/samvera/hyrax/issues/1678 for what else is needed to complete show page work.

### CODE REVIEW

File: app/views/hyrax/dashboard/collections/_show_document_list_row.html.erb [hunk 1]
```diff
@@ -20,6 +20,7 @@
     </div>
   </td>
   <td class="text-center"><%= document.date_uploaded %> </td>
+  <td class="text-center">[Owner Goes Here]</td>
```
Comment (jcoyne) [inline app/views/hyrax/dashboard/collections/_show_document_list_row.html.erb]: Does this need to be completed?

File: app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
-          data: { turbolinks: false } %>
-      <% end %>
+    <div class="text-right">
+      <div>
+        <div class="sr-only"><% t('hyrax.collection.actions.nest_collections.desc') %></div>
+        <%= button_tag '',
+              class: 'btn btn-primary add-subcollection',
+              onclick: "$('#add-subcollection-modal-#{presenter.id}').modal('show');",
+              title: t('hyrax.collection.actions.nest_collections.desc'),
+              type: 'button',
```
Comment (jcoyne) [inline app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb]: Necessary?

File: app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb [hunk 3]
```diff
@@ -1,26 +1,30 @@
...
-        <%= link_to t('hyrax.collection.actions.add_new_nested_collection.label'),
-          hyrax.dashboard_create_subcollection_under_path(parent_id: presenter.id),
-          title: t('hyrax.collection.actions.add_new_nested_collection.desc'),
-          class: 'btn btn-default pull-right',
-          data: { turbolinks: false } %>
-      <% end %>
+    <div class="text-right">
+      <div>
+        <div class="sr-only"><% t('hyrax.collection.actions.nest_collections.desc') %></div>
+        <%= button_tag '',
```
Comment (jcoyne) [inline app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb]: What is the purpose of this argument when passing a block?

File: app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb [hunk 4]
```diff
@@ -1,26 +1,30 @@
...
+              class: 'btn btn-primary add-subcollection',
+              onclick: "$('#add-subcollection-modal-#{presenter.id}').modal('show');",
```
Comment (jcoyne) [inline app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb]: It would be good to move this to a JS file

File: app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb [hunk 5]
```diff
@@ -1,26 +1,30 @@
...
+      </div>
+      <div>
+        <!-- The user should must have the ability to create a new collection of parent's type -->
+        <% if presenter.user_can_create_new_nest_collection? %>
+          <div class="sr-only"><% t('hyrax.collection.actions.add_new_nested_collection.desc') %></div>
+          <%= link_to t('hyrax.collection.actions.add_new_nested_collection.label'),
+            hyrax.dashboard_create_subcollection_under_path(parent_id: presenter.id),
+            title: t('hyrax.collection.actions.add_new_nested_collection.desc'),
+            class: 'btn btn-link side-arrows',
+            data: { turbolinks: false } %>
```
Comment (jcoyne) [inline app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb]: Why is this necessary?

File: app/views/hyrax/dashboard/collections/_sort_and_per_page.html.erb [hunk 6]
```diff
@@ -10,23 +10,25 @@
...
-               </fieldset>
-               <%= render 'view_type_group' %>
-             </div>
-         <% end %>
+      <%= form_tag dashboard_collection_path(collection), method: :get, class: 'per_page form-horizontal' do %>
+         <div class="form-group form-group-lg">
+           <fieldset class="col-sm-9">
+             <legend class="sr-only"><%= t('hyrax.sort_label') %></legend>
+             <%= label_tag(:sort, 'Sort By:') %>
+             <%= select_tag(:sort, options_for_select(active_sort_fields, h(params[:sort]))) %>
```
Comment (jcoyne) [inline app/views/hyrax/dashboard/collections/_sort_and_per_page.html.erb]: Why are you calling h()?

File: app/views/hyrax/dashboard/collections/_sort_and_per_page.html.erb [hunk 7]
```diff
@@ -10,23 +10,25 @@
...
+             <%= label_tag(:per_page) do %>
+                 Show <%= select_tag(:per_page, options_for_select(['10', '20', '50', '100'], h(params[:per_page])), title: "Number of results to display per page") %>
+                 per page
+             <% end %>
+             <%= render_hash_as_hidden_fields(search_state.params_for_search.except(:per_page, :sort)) %>
+             &nbsp;&nbsp;&nbsp;
```
Comment (jcoyne) [inline app/views/hyrax/dashboard/collections/_sort_and_per_page.html.erb]: Are these required? Should it be CSS instead?

File: app/views/hyrax/dashboard/collections/_sort_and_per_page.html.erb [hunk 8]
```diff
@@ -10,23 +10,25 @@
...
+             <button class="btn btn-info"><span class="glyphicon glyphicon-refresh"></span> Refresh</button>
```
Comment (jcoyne) [inline app/views/hyrax/dashboard/collections/_sort_and_per_page.html.erb]: I18n this string please

File: app/views/hyrax/dashboard/collections/show.html.erb [hunk 9]
```diff
@@ -4,63 +4,80 @@
...
-    <% end %>
-  </div>
-  <div class="col-xs-6 col-md-5 col-lg-6 pull-right">
-    <%# TODO: leaving this as it was causes rerouting to the public show page. needs work %>
-    <%= render 'hyrax/collections/search_form', presenter: @presenter %>
-  </div>
-</div>
+            <% #TODO: Should the display of this be conditional? %>
+            <section class="parent-collection-wrapper">
+              <h4>[TODO: Wire up] Parent Collection</h4>
```
Comment (jcoyne) [inline app/views/hyrax/dashboard/collections/show.html.erb]: Need finishing?

Comment (adamjarling) [reply app/views/hyrax/dashboard/collections/show.html.erb]: Ah my bad.  Fixing now.

File: app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb (hunk 3 above)
Comment (adamjarling) [reply app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb]: For this, I don't know.  I just picked up this code and moved it.

**Comment by adamjarling** [comment]
@jcoyne For my work on this ticket, I just took existing code put into into bootstrap panels so it doesn't look like this (attached).

collections-show-page-bad

So, I might not have answers to some of these comments on why certain things were done.  No clue.  But I can fix some of the stuff you mentioned, I think.

File: app/views/hyrax/dashboard/collections/_show_document_list_row.html.erb (hunk 1 above)
Comment (adamjarling) [reply app/views/hyrax/dashboard/collections/_show_document_list_row.html.erb]: @elrayle suggested creating a new ticket to get this in place.  https://github.com/samvera/hyrax/issues/2478

File: app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb (hunk 5 above)
Comment (adamjarling) [reply app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb]: I'm just going to remove it.

**Comment by jcoyne** [comment]
@adamjarling My concern is that none of this code is in the master branch and I would like it cleaned up prior to getting it into master.  Where do you think is the best place for that to happen?

File: app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb (hunk 5 above)
Comment (laritakr) [reply app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb]: This is the button option to create a new subcollection as a child of the collection. It is consistent with the ability to create a new nested work on the collection.

File: app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb (hunk 2 above)
Comment (laritakr) [reply app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb]: This is the button option to nest an existing collection as a child of the collection. It is consistent with the ability to nest an existing work on the collection.

Comment (laritakr) [reply app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb]: This is the option to add an existing collection as a nested collection (I believe). The placement of these options are consistent with the corresponding options available for adding works to the collection.

Comment (adamjarling) [reply app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb]: I removed `type: 'button'` and it displays still.  Maybe just redundant.

**Comment by adamjarling** [comment]
@jcoyne Yea, I get it.  All the items in the comments should now be addressed.

**Comment by adamjarling** [comment]
@elrayle Looks like maybe your review got dismissed?   If it's ok and you approve, mind merging in case I'm offline?
